app/src/org/odk/collect/android/widgets/QuestionWidget.java

```diff
@@ -649,4 +649,7 @@ public Spannable forceMarkdown(String text){
     public Spannable stylize(String text){
         return MarkupUtil.styleSpannable(getContext(), text);
     }
+
+    public void setBinaryData(Object answer) {
+    }
```

Comment:
Should just add a comment that this gets implemented in the subclasses that need it.